puppet/modules/jenkins_job_builder/files/centos.org/jobs/foreman-test.yml

```diff
@@ -0,0 +1,102 @@
+---
+- job-template:
+    name: foreman-{version}-{distro}-{action}-test
+    project-type: 'pipeline'
+    sandbox: true
+    parameters:
+      - string:
+          name: expected_version
+          default: 'UNKNOWN'
```

Comment:
this default will break nightly tests that you now have folded into the same test definition.

Reply:
What part breaks? (maybe I can fix it?)

Reply:
We have a test in `forklift` that verifies that the expected version is the one installed:
https://github.com/theforeman/forklift/blob/42a15959db28e1bf6b51e3982952094ec97dff70/bats/fb-test-foreman.bats#L6-L10

It only triggers if the environment variable is actually set, but my reading of this line and https://github.com/theforeman/foreman-infra/pull/1355/files#diff-0bccc46806d9b5ebbc8e2aacf5a8717eR1 suggests it will be always set.

Reply:
Perhaps we just set this for nightly to the version that is currently set in develop? Unify that test across releases and nightly.

Reply:
This is also what @ekohl suggested in https://github.com/theforeman/foreman-infra/pull/1355#discussion_r419144553 -- sounds good to me